README.md

```diff
@@ -2,6 +2,14 @@
 [![GitHub license](https://img.shields.io/badge/license-LGPL-blue.svg)](https://raw.githubusercontent.com/PedestrianDynamics/jupedsim/master/LICENSE)
 ![PyPI - Python Version](https://img.shields.io/pypi/pyversions/jupedsim)
 ![PyPI - Version](https://img.shields.io/pypi/v/jupedsim)
+[![Build CI wheels](https://github.com/PedestrianDynamics/jupedsim/actions/workflows/cibuildwheel.yml/badge.svg)](https://github.com/PedestrianDynamics/jupedsim/actions/workflows/cibuildwheel.yml)
+[![Documentation Status](https://img.shields.io/badge/docs-up--to--date-brightgreen)](https://jupedsim.org/stable)
```

Comment:
A successful documentation build is part of our CI. This badge is superfluous imo.